src/set_theory/cardinal/basic.lean

```diff
@@ -474,6 +476,8 @@ instance : canonically_ordered_comm_semiring cardinal.{u} :=
     by simpa only [mul_def, mk_eq_zero_iff, is_empty_prod] using id,
   ..cardinal.comm_semiring, ..cardinal.partial_order }
 
+protected theorem not_lt_zero (c : cardinal) : ¬ c < 0 := not_lt_bot
```

Comment:
Doesn't [`not_lt_zero'`](https://leanprover-community.github.io/mathlib_docs/algebra/order/with_zero.html#not_lt_zero') apply to `cardinal` already?

Reply:
Cardinals are not a linear ordered commutative monoid. We have `ℵ₀ * 2 ≤ ℵ₀ * 1` (in fact equality) but not `2 ≤ 1`.

Reply:
I didn't say `linear_ordered_cancel_comm_monoid`, but `linear_ordered_comm_monoid`.

Reply:
That's true, we were missing that instance. I've added it separately.

Reply:
Can you remove this lemma then?